addons/hr_holidays/tests/test_avatar_card_tour.py

```diff
@@ -0,0 +1,185 @@
+        })
+        leave_type = (
+            cls.env["hr.leave.type"]
+            .with_company(company_2)
+            .create(
+                {
+                    "name": "Time Off multi company",
+                    "company_id": company_2.id,
+                    "time_type": "leave",
+                    "requires_allocation": False,
+                }
+            )
+        )
+        partner = (
+            cls.env["res.partner"]
+            .with_company(company_2)
+            .create(
+                {
+                    "name": "Test User Partner",
+                    "company_id": company_2.id,
+                    "email": "test@test.com",
+                    "phone": "123456789",
+                }
+            )
+        )
+        cls.test_user = (
+            cls.env["res.users"]
+            .with_company(company_2)
+            .create(
+                {
+                    "name": "Test User",
+                    "login": "test_user",
+                    "password": "test_password",
+                    "company_id": company_2.id,
+                    "company_ids": [Command.set([company_2.id])],
+                    "partner_id": partner.id,
+                }
+            )
+        )
+
+        cls.test_user.partner_id.write(
+            {
+                "user_id": cls.test_user.id,
+            }
+        )
+        department = (
+            cls.env["hr.department"]
+            .with_company(company_2)
+            .create(
+                {
+                    "name": "Test Department",
+                    "company_id": company_2.id,
+                }
+            )
+        )
+        job = (
+            cls.env["hr.job"]
+            .with_company(company_2)
+            .create(
+                {
+                    "name": "Test Job Title",
+                    "company_id": company_2.id,
+                }
+            )
+        )
+        other_partner = (
+            cls.env["res.partner"]
+            .with_company(company_2)
+            .create(
+                {
+                    "name": "Test Other Partner",
+                    "company_id": company_2.id,
+                    "phone": "987654321",
+                }
+            )
+        )
+        test_employee = (
+            cls.env["hr.employee"]
+            .with_company(company_2)
+            .create(
+                {
+                    "name": "Test Employee",
+                    "user_id": cls.test_user.id,
+                    "company_id": company_2.id,
+                    "department_id": department.id,
+                    "job_id": job.id,
+                    "address_id": other_partner.id,
+                    "work_email": "test_employee@test.com",
+                }
+            )
+        )
+        cls.test_user.write(
+            {
+                "employee_ids": [Command.link(test_employee.id)],
+            }
+        )
```

Comment:
The goal is to have fewer lines possible. Most of those could be written in single line (or at least much less lines), even if using the formatter.

Also it's not always necessary, but if the data can't fit on a single line, sometimes making separate variable for the data helps a lot.